### DISCUSSION TITLE
Event uses invalid property for type

### DISCUSSION THREAD

**Discussion by tholzheim** [SmartDataAnalytics/OpenResearch#142] [Q&A]
The Event type is stored in the Property [Property:Type](https://www.openresearch.org/wiki/Property:Type). But this property seems to be occupied by an extension and therefore should not be used.
This might also be the issue why the type of an event can not be queried.

> [Propert:Event type](https://www.openresearch.org/wiki/Property:Event_type) seems to be the intended property for event types

### Example
The following query should return the title and the type of events
```
{{#ask: [[isA::Event]]|mainlabel=Event|?Type|?Title|format=table}}
```

Event | Type | Title
-- | -- | --
[" DBKDA 2021"](https://www.openresearch.org/wiki/%22_DBKDA_2021%22) |   | The Thirteenth International Conference on Advances in Databases, Knowledge, and Data Applications
["ENERGY 2021"](https://www.openresearch.org/wiki/%22ENERGY_2021%22) |   | The Eleventh International Conference on Smart Grids, Green Communications and IT Energy-aware Technologies
["ICAS 2021"](https://www.openresearch.org/wiki/%22ICAS_2021%22) |   | The Seventeenth International Conference on Autonomic and Autonomous Systems

But the type is missing eventhough it is defined
```
{{Event
|Acronym=" DBKDA 2021"
|Title=The Thirteenth International Conference on Advances in Databases, Knowledge, and Data Applications
|Type=Conference
|Field=Advances in Databases, Knowledge, and Data Applications
|Start date=2021/05/30
|End date=2021/06/03
|Submission deadline=2021/03/20
|Homepage=https://www.iaria.org/conferences2021/DBKDA21.html
|City=Valencia
|State=Valencia
|Country=Spain
}}
```

**Comment by WolfgangFahl** [comment]
Well observed. Please open a Ticket for this.
